src/presentations/Footer.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Handlebars from "handlebars";
+const hbr = `<div class="full-width-footer footer-area"><div class="row">
+<footer class="space-footer">
+Copyright @ 2011-2018 Sabka Bazaar Grocery Supplies Pvt. Ltd.
+</footer></div><div>
+`;
+
+const template = Handlebars.compile(hbr);
```

Comment:
dont wrap footer inside div
what is the purpose of handlebar here if you are directly assigning the html template to the dom using dangerouslySetInnerHtml?

Reply:
The point is, handlebar usage seems irrelevant here